packages/patternfly-4/react-core/src/components/Select/SelectMenu.tsx

```diff
@@ -45,21 +47,23 @@ export class SelectMenu extends React.Component<SelectMenuProps> {
     selected: '',
     maxHeight: '',
     sendRef: Function.prototype,
-    keyHandler: Function.prototype
+    keyHandler: Function.prototype,
+    isCustomContent: false
   };
 
   extendChildren() {
     const { children, isGrouped } = this.props;
+    const childrenArray = children as React.ReactElement[];
```

Comment:
Can you do this instead const childrenArray: React.ReactElement[] = children as React.ReactElement[];    ... so it's not implicit? Or will this not work with here?